src/AzSK.ADO/Framework/Abstracts/ServicesSecurityStatus.ps1

```diff
@@ -533,7 +533,9 @@ class ServicesSecurityStatus: ADOSVTCommandBase
 
 		# append summary counts for bug logging & append control results with bug logging data
 		if($this.IsBugLoggingEnabled){
-			$partialScanMngr.CollateBugSummaryData($result);
+			if(!$this.invocationContext.BoundParameters["AutoCloseBugs"]){
```

Comment:
at line 37 IsBugLoggingEnabled  is set to true only when autoclosebugs/autobuglog is used in command
Do we need to check the condition again?

Reply:
In  AutoCloseBugs there will be not active/resolved/new bugs. So that function will do nothing. It will also populate a static variable with all failed/verify controls which we don't require as bug logging didnt happen